### ISSUE TITLE
inquiry

### ISSUE THREAD

**Issue by zhang-glitch** [theajack/disable-devtool#82]
Hello, author. At present, it is not only effective to monitor the width and height of the visual area and check the performance by printing. Nothing else works, okay?

**Comment by theajack** [comment]
Yes, I think so. As browsers are upgraded, many strategies become ineffective. @zhang-glitch

**Comment by theajack** [comment]
However, some strategies are still effective for different browsers and different versions, so it is reasonable to make redundant judgments.
